lib/modules/datasource/sbt-plugin/index.ts

```diff
@@ -9,10 +9,9 @@ import { SbtPackageDatasource } from '../sbt-package';
 import { getLatestVersion, parseIndexDir } from '../sbt-package/util';
 import type { GetReleasesConfig, ReleaseResult } from '../types';
 
-export const SBT_PLUGINS_REPO =
-  'https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases';
+export const SBT_MVN_REPO = 'https://repo1.maven.org/maven2';
```

Comment:
how do you verified that all existing versions are migrated to maven.org?

Reply:
Good point. I did not notice that older plugins aren't present in the newer repo. I've put the old URL back in, but I've made the maven URL the first in the list.